Test/VimCoreTest/InterpreterTest.cs

```diff
@@ -1316,6 +1316,17 @@ public void RHSCanBeArbitraryExpression()
                 AssertValue("x", "Hello, world!");
             }
 
+            [WpfFact]
+            public void RHSCanBeEnvironmentVariable()
+            {
+                Create("");
+                var variable = "magic";
+                var value = "xyzzy";
+                Environment.SetEnvironmentVariable(variable, value);

```

Comment:
Should unset this at the end of the test to make sure another test doesn't accidentally depend on it.